src/core/agent/services/ipexCommunicationService.ts

```diff
@@ -223,6 +223,7 @@ class IpexCommunicationService extends AgentService {
       schema: {
         name: schemaKeri.title,
         description: schemaKeri.description,
+        attributes: attributes,
```

Comment:
`attributes` should be top level in the JSON object returned, instead of inside `schema`